SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -148,7 +175,40 @@ public static void ForceVoltage(this DCPowerSessionsBundle sessionsBundle, PinSi
         }
 
         /// <summary>
-        /// Forces voltage and specifies symmetric current limit.
+        /// Forces voltage and specifies current limit.
+        /// </summary>
+        /// <param name="sessionsBundle">The <see cref="DCPowerSessionsBundle"/> object.</param>
+        /// <param name="settings">The settings to use.</param>
+        /// <param name="waitForSourceCompletion">Setting this to True will wait until sourcing is complete before continuing, which includes the set amount of source delay.
+        /// Otherwise, the source delay amount is not directly accounted for by this method and the WaitForEvent must be manually invoked in proceeding code.</param>
+        public static void ForceVoltage(this DCPowerSessionsBundle sessionsBundle, DCPowerSourceSettings settings, bool waitForSourceCompletion = false)
+        {
+            sessionsBundle.Do(sessionInfo =>
+            {
+                settings.OutputFunction = DCPowerSourceOutputFunction.DCVoltage;
+                sessionInfo.Force(settings, sessionInfo.AllChannelsString, waitForSourceCompletion);
+            });
+        }
+
+        /// <summary>
+        /// Forces voltage and specifies current limit.
```

Comment:
Suggest changing to: "Forces voltage using specified site-unique source settings."